client/src/components/puzzleShow.js

```diff
@@ -1,7 +1,115 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom"
+import ErrorList from "./ErrorList.js"
+import translateServerErrors from "../services/translateServerErrors.js"
+import BoxTile from "./BoxTile"
 
 const puzzleShow = (props) => {
+  const [puzzle, setPuzzle] = useState([])
 
+  const { id } = useParams()
+
+  const getPuzzle = async () => {
+    try{
+      const response = await fetch(`/api/v1/puzzle/${id}`)
+      if(!response.ok){
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const body = await response.json()
+      setPuzzle(body.puzzle)
+
+    } catch(error) {
+      console.error(`Error in fetch: ${error.message}`)
+    }
+  }
+
+  useEffect(() => {
+    getPuzzle()
+  }, [])
+
+  const handleInputChange = (event) => {
+    setPuzzle({
+      ...puzzle,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+  }
+
+  
+  let boxOne
+  let boxTwo
+  let boxThree
+  let boxFour
+  let boxFive
+  let boxSix
+  let boxSeven
+  let boxEight
+  let boxNine
+  if(puzzle.boxOne !== undefined) {
+    boxOne = 
+    <BoxTile box={"box1"} puzzle={puzzle.boxOne} handleInputChange={handleInputChange}/>
```

Comment:
for the `box` prop, convention is to drop the `{ }` when passing strings 
```
<BoxTile box="box1" puzzle={puzzle.boxOne} handleInputChange={handleInputChange}/>
```
And further to make this multiline once you have more than two props 
```
<BoxTile 
   box={"box1"} 
   puzzle={puzzle.boxOne} 
   handleInputChange={handleInputChange}
/>
```
but the multiline I can leave to your disgression, its pretty okay